src/block.rs

```diff
@@ -0,0 +1,25 @@
+use crate::PeerId;
+
+use fuel_tx::Transaction;
```

Comment:
What do people think about making the transaction type a generic trait, rather than directly depending on fuel-tx?

Reply:
That sounds like a good idea